CHANGELOG.md

```diff
@@ -6,6 +6,13 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 
 ## [Unreleased]
 
+## [15.1.0]
+### Changed
+- Update constants.ts to add a BSC url for smart transactions ([#483](https://github.com/MetaMask/smart-transactions-controller/pull/483))
+
+### Removed
+- Remove unnecessary `events` dependency ([#473](https://github.com/MetaMask/smart-transactions-controller/pull/473))
```

Comment:
Nit: In the future, please put changes like this under "Changed". "removed" is for "removed features', this is not a removed feature.

See the keepachangelog docs linked at the top of the changelog for more information